examples/angularjs_require/js/directives/todoEscape.js

```diff
@@ -0,0 +1,16 @@
+/*global define*/
+define(['app'], function (app) {
+	'use strict';
+
+	app.directive('todoEscape', function () {
+		var ESCAPE_KEY = 27;
+
+		return function (scope, elem, attrs) {
+			elem.bind('keydown', function (event) {
+				if (event.keyCode === ESCAPE_KEY) {
```

Comment:
since we are setting this binding outside of angular's knowledge we are going to want to clean it up via 

`scope.$on('$destroy', function() {`

otherwise we are going to get into leak hell

unlessssss
https://docs.angularjs.org/api/ng/function/angular.element#events

is magically cleaning this up for us, thoughts @arthurvr ?